rpi5.yaml

```diff
@@ -0,0 +1,216 @@
+desc: "Raspberry 5 with xen dom0less"
+min_ver: "0.22"
+
+
+variables:
+  # Common build configuration
+
+  YOCTOS_WORK_DIR: "yocto"
+
+  DOM0_OS: "zephyr"
+
+  # Raspberry 5 build configuration
+
+  RPI_MACHINE: "raspberrypi5"
+  RPI_SOC_FAMILY: "bcm2712"
+
+  ZEPHYR_DOM0_DIR: "zephyr"
+  ZEPHYR_DOM0_MACHINE: "rpi_5_xen_dom0"
+  ZEPHYR_DOM0_TARGET: "zephyr/samples/hello_world"
+  ZEPHYR_DOM0_BUILD_DIR: "build-dom0"
+
+  DOMD_BUILD_DIR: "build-domd"
+  XT_XEN_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-xen.dtb"
+  XT_DOMD_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-domd.dtb"
+
+  XT_KERNEL_BRANCH: "rpi-6.6.y"
+  XT_KERNEL_REV: "${AUTOREV}"
+
+  XT_DOMD_IMAGE: "rpi5-image-minimal-domd"
+
+common_data:
+  # Sources used by domd
+  domd_sources: &DOMD_DOMU_SOURCES
+    - type: git
+      url: "git://git.yoctoproject.org/poky"
+      rev: "scarthgap"
+    - type: git
+      url: "git://git.openembedded.org/meta-openembedded"
+      rev: "scarthgap"
+    - type: git
+      url: "git://git.yoctoproject.org/meta-virtualization"
+      rev: "scarthgap"
+    - type: git
+      url: "https://github.com/dsemenets/meta-xt-rpi5.git"
+      rev: "linux"
+    - type: git
+      url: "https://github.com/dsemenets/meta-xt-rpi5-prod-devel.git"
+      rev: "init_new"
+    - type: git
+      url: "git://git.yoctoproject.org/meta-selinux"
+      rev: "master"
+
+  # Zephyr sources
+  zephyr_sources: &ZEPHYR_SOURCES
+    - type: west
+      url: "https://github.com/GrygiriiS/zephyr.git"
+      rev: "zephyr-v3.6.0-xt-rpi5-dom0-wip-dom0"
+
+  # DomD configuration options for all yocto-based domains
+  domd_conf: &DOMD_CONF
+    - [SSTATE_DIR, "${TOPDIR}/../../../common_data/sstate"]
+    - [DL_DIR, "${TOPDIR}/../../../common_data/downloads"]
+
+    # Skip warning about missing "virtualization" distro feature
+    - [SKIP_META_VIRT_SANITY_CHECK, "1"]
+
+    # Init manager
+    - [INIT_MANAGER, "systemd"]
+
+    # Do not install kernel image to rootfs to decrease initrd size
+    - ["RDEPENDS_${KERNEL_PACKAGE_NAME}-base", ""]
+
+    # Remove features that we are not using
+    - [
+        DISTRO_FEATURES:remove,
+        "x11 gtk gobject-introspection-data nfc irda zeroconf 3g sysvinit
+         acl alsa argp pcmcia usbgadget opengl ptest multiarch wayland
+         vulkan sysvinit",
+      ]
+
+components:
+  dom0:
+    default: true
+    build-dir: "%{ZEPHYR_DOM0_DIR}"
+    sources:
+      - *ZEPHYR_SOURCES
```

Comment:
This is the only place where `ZEPHYR_SOURCES` is used. Why didn't put actual list of sources here?

Reply:
To keep all sources in one place

Reply:
But why you couldn't put them to dom0, where they should belong? We use `&`/`*` syntax only when there is a shared part of configs/sources. But in this .yaml you define two domains: dom0 and domd, which share exactly nothing.

Reply:
Fixed